### ISSUE TITLE
Dont use java/com/example

### ISSUE THREAD

**Issue by Fyoncle** [First-Name-Last-Name/Swimming-Horses#2]
Yeah, java/com/example is kinda weird and should be changed to src/main/java/net/FirstNameLastName

**Comment by First-Name-Last-Name** [comment]
Alright, thanks for the heads up. You think I can change it to com.github.first-name-last-name? I read that that’s the convention for when you don’t have any other domains you control

**Comment by Fyoncle** [comment]
> Alright, thanks for the heads up. You think I can change it to com.github.first-name-last-name? I read that that’s the convention for when you don’t have any other domains you control

Not really, you can use net, com, or anything you want, but if you prefer com thats also good but dont use java example
